definitions/variable/macro-economy/gdp.yaml

```diff
@@ -1,6 +1,25 @@
 - GDP|MER:
     definition: GDP converted to USD at market exchange rate (MER)
     unit: billion USD_2010/yr
+    tier: 1
 - GDP|PPP:
     definition: GDP converted to USD using purchasing power parity (PPP)
     unit: billion USD_2010/yr
+    tier: 1
+- Value Added|{Econ sectors}:
+    description: value added of the {Econ sectors}
+    unit: billion USD_2010/yr
+    tier: 1
```

Comment:
here there are 2 entries for tier. If `tier: {Econ sectors}`, then it is extracted from the tag file?